ts/Series/Column/ColumnSeries.ts

```diff
@@ -1115,7 +1115,7 @@ class ColumnSeries extends LineSeries {
                 verb = graphic && chart.pointCount < animationLimit ?
                     'animate' : 'attr';
 
-            if (isNumber(plotY) && point.y !== null) {
+            if (isNumber(plotY) && point.y !== null && point.visible !== false) {
```

Comment:
For the sake of consistency, I think we should do this the same way as we do with pie series. We may want to refactor this repeated pattern in the future.

https://github.com/highcharts/highcharts/blob/35163c02456af62b4f26a93a068510844ae7a25d/ts/Series/Pie/PieSeries.ts#L951-L953